app/controllers/concerns/reauthentication_required_concern.rb

```diff
@@ -24,6 +35,14 @@ def prompt_for_current_password
     redirect_to user_password_confirm_url
   end
 
+  def prompt_for_second_factor
+    store_location(request.url)
+    user_session[:context] = 'reauthentication'
+    user_session[:factor_to_change] = factor_from_controller_name
```

Comment:
no action needed in this PR but 😭 I really with we'd fixed the i18n bug in this LG-5701

Reply:
oh oops, this was supposed to be dropped in the new process

Reply:
Removed